vis/xAPISG-CorrectIncorrectPerPlayer.ipynb

```diff
@@ -45,90 +54,54 @@
     "            total_false += results.count(\"False\") +results.count(False)\n",
     "            if total_true+total_false>total_max:\n",
     "                total_max=total_true+total_false\n",
-    "        true_per_player.append(total_true)\n",
-    "        false_per_player.append(total_false)\n",
+    "        true_false_per_player[player][\"true\"]=total_true\n",
+    "        true_false_per_player[player][\"false\"]=total_false\n",
     "    total_max+=0.5\n",
```

Comment:
Why this?

Reply:
I put all in a dictionary to get in memory the association player, (true, false) response and the value number to order all after with function "orderDict"